kuma/static/styles/libs/prism/prism-line-numbers.css

```diff
@@ -37,4 +37,8 @@ pre.line-numbers > code {
 			display: block;
 			padding-right: 0.8em;
 			text-align: right;
-		}
\ No newline at end of file
+		}
+
+.no-line-numbers .line-numbers-rows {
+    display:none;
+}
```

Comment:
Because `.line-numbers-rows` only has rules applied when it's a child of an element with the `.line-numbers` class (line 11 above), it's probably more clear to do something like:

```
// new selector to replace the .no-line-numbers selector
.line-numbers-rows {
    display: none;
}

// existing selector
.line-numbers .line-numbers-rows {
    display: block;
    // all the other pre-existing rules here
}
````